workflow/tests/util.rb

```diff
@@ -14,6 +14,7 @@ def run_simulation_tests(xmls)
 
     next unless xml.include?('sample_files') || xml.include?('real_homes') # Exclude e.g. ASHRAE 140 files
     next if xml.include? 'base-bldgtype-mf-whole-building' # Already has multiple dwelling units
+    next if xml.include? 'max-power-ratio-schedule' # Unit multiplier not supported yet
```

Comment:
Note that the 10x case tests two things -- 1) running a whole SFA/MF building (2x) and 2) using unit multipliers (5x), which combine to 10x.

If unit multipliers is not allowed, that's fine, but we should still test running a whole SFA/MF building without them. So please remove this line and handle it the same way we handle [dehumidifiers and GSHPs](https://github.com/NREL/OpenStudio-HPXML/blob/4f60b2e19cb413803022cbafef31d28e083f820e/workflow/tests/util.rb#L42-L47), where we only exclude the unit multipliers from the test.